powerVCF.psm1

```diff
@@ -1297,32 +1297,43 @@ Function Get-VCFCredential {
 <#
     .SYNOPSIS
     Connects to the specified SDDC Manager and retrieves a list of credentials.
+    Supported resource types are: PSC, VCENTER, ESXI, NSX_MANAGER, NSX_CONTROLLER, BACKUP
+    Please note: if you are requesting credentials by resource type then the resource name parameter (if passed) will be ignored (they are mutually exclusive)
 
     .DESCRIPTION
     The Get-VCFCredential cmdlet connects to the specified SDDC Manager and retrieves a list of
     credentials. A privileged user account is required.
 
     .EXAMPLE
-    PS C:\> Get-VCFCredential -privilegedUsername sec-admin@rainpole.local
-	-privilegedPassword VMw@re1!
+    PS C:\> Get-VCFCredential -privilegedUsername sec-admin@rainpole.local -privilegedPassword VMw@re1!
     This example shows how to get a list of credentials
 
+    .EXAMPLE
```

Comment:
Maybe use a different resourceType as PSC will go away in the next major release!

Reply:
>PSC will go away in the next major release
well I'm not aware of what's going ... for now I can only develop against what's available
that said, fixing that will be quick :-)